google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -100,7 +101,7 @@ public Bucket createBucketWithStorageClassAndLocation(String bucketName) {
     // [START createBucketWithStorageClassAndLocation]
     Bucket bucket = storage.create(BucketInfo.newBuilder(bucketName)
         // See here for possible values: http://g.co/cloud/storage/docs/storage-classes
-        .setStorageClass("COLDLINE")
+        .setStorageClass(StorageClass.COLDLINE)
```

Comment:
According to the URL above this line, the storage classes are in lower case. Do the integration tests pass?